src/config.rs

```diff
@@ -143,6 +145,16 @@ pub struct Config {
     ///
     /// [`PROBE`]: static@PROBE
     pub format_registry: &'static Probe,
+    #[cfg(feature = "driver")]
+    /// The Sender for a channel that will run the destructor of possibly blocking values.
+    ///
+    /// If not set, a thread will be spawned to perform this, but it is recommended to create
+    /// a long running thread instead of relying on a per-driver thread.
+    ///
+    /// Note: When using [`Songbird`] this is overwritten automatically by it's disposal thread.
+    ///
+    /// [`Songbird`]: crate::Songbird
+    pub disposer: Option<Sender<DisposalMessage>>,
```

Comment:
`Config` is the right place for this to be set, but I'm unhappy with the type `Sender` being exposed since that now makes flume upgrades into breaking changes.

Is the right choice maybe to make an opaque newtype to handle this?
```rust
pub struct DisposalThread(Sender<DisposalMessage>);
```
I guess this has a really obvious `Default` impl which you can lean on elsewhere, and still allows for users to set up whatever number and allocation of disposal threads they want.

You might be able to define a `DisposalThread::with_fn` as well to take a closure/fn, but I don't think that's a good idea as users shouldn't be able to access `InnerTracks` etc. anyway.

Reply:
Do you mean by the default impl to spawn the thread in that? If so, that solves the issue I had in mind of the disposal sender being used for stuff other than disposing....

Reply:
Exactly, so something like:

```rust
impl Default for DisposalThread {
    pub fn default() -> Self {
        Self(disposal::run)
    }
}
```

You'd likely need a `pub(crate)` on the inner `Sender` field, after some reflection...

Reply:
Done! Didn't need to pub(crate) it and cleaned up some stuff in the process